### PR TITLE
fix:[DOCS] Add proper README

### LINKED ISSUE #7: [DOCS] Add proper README
Add proper README with more info, badges, etc.

### PR DESCRIPTION
**Pull request by Pericles001**
resolves #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+
+  
+*  Netlify will post a deploy link as a comment in your PR thread.
 
 
 
 ## Deploy Status
 [![Netlify Status](https://api.netlify.com/api/v1/badges/f7ed0fa8-4cf0-429b-bea5-95d9c83b8391/deploy-status)](https://app.netlify.com/sites/free-hosting-via-pr/deploys)
+
+## Aknowledgements
```
Comment (KendallDoesCoding) [inline README.md:64]:
```suggestion
## Acnowledgements
```

fix typo

File: README.md [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+
+...
```
Comment (KendallDoesCoding) [inline README.md]: KendallDoesCoding - built software
Netlify - using their software.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KendallDoesCoding) [reply README.md]: @Pericles001 Please add the needful and will merge.

Comment (Pericles001) [reply README.md]: I will do that soon. Thanks for reviewing.

Comment (KendallDoesCoding) [reply README.md]: Thanks!

Comment (Pericles001) [reply README.md]: It is done.

**Review by KendallDoesCoding — APPROVED** [review]
LGTM ✔️

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution! Merging.

**Comment by KendallDoesCoding** [comment]
Thanks for your contribution! Merging.

**Comment by Pericles001** [comment]
> Thanks for your contribution! Merging.

You're welcome!

**Pull request merged**